doc/orchestrator.rst

```diff
@@ -165,3 +165,541 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+We use the ``ConfigOptions.create_from_environment()``
+function to create three instances of ``ConfigOptions``,
+with one instance associated with each launched ``Orchestrator``.
+Most importantly, to associate each launched Orchestrator to a ConfigOptions object,
+the ``create_from_environment()`` function requires specifying the unique database identifier
+argument named `db_identifier`.
+
+For the single-sharded database:
+
+.. code-block:: python
+
+  single_shard_config = ConfigOptions.create_from_environment("single_shard_db_identifier")
+
+For the multi-sharded database:
+
+.. code-block:: python
+
+  multi_shard_config = ConfigOptions.create_from_environment("multi_shard_db_identifier")
+
+For the colocated database:
+
+.. code-block:: python
+
+  colo_config = ConfigOptions.create_from_environment("colo_db_identifier")
+
+Step 2: Initialize the Client Connections
+"""""""""""""""""""""""""""""""""""""""""
+Now that we have three ``ConfigOptions`` objects, we have the
+tools necessary to initialize three SmartRedis clients and
+establish a connection with the three databases.
+We use the SmartRedis ``Client`` API to create the client instances by passing in
+the ``ConfigOptions`` objects and assigning a `logger_name` argument.
+
+Single-sharded database:
+
+.. code-block:: python
+
+  app_single_shard_client = Client(single_shard_config, logger_name="Model: multi shard logger")
+
+Multi-sharded database:
+
+.. code-block:: python
+
+  app_multi_shard_client = Client(multi_shard_config, logger_name="Model: single shard logger")
+
+Colocated database:
+
+.. code-block:: python
+
+  colo_client = Client(colo_config, logger_name="Model: colocated logger")
+
+Retrieve Data and Store Using SmartRedis Client Objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To confirm a successful connection to each database, we will retrieve the tensors
+that we plan to store in the python driver script. After retrieving, we
+store both tensors in the colocated database.
+The ``Client.get_tensor()`` method allows
+retrieval of a tensor. It requires the `name` of the tensor assigned
+when sent to the database via ``Client.put_tensor()``.
+
+.. code-block:: python
+
+  val1 = app_single_shard_client.get_tensor("tensor_1")
+  val2 = app_multi_shard_client.get_tensor("tensor_2")
+  app_single_shard_client.log_data(LLInfo, f"The colocated db tensor is: {val1}")
+  app_multi_shard_client.log_data(LLInfo, f"The clustered db tensor is: {val2}")
+
+Later, when you run the experiment driver script the following output will appear in ``tutorial_model.out``
+located in ``getting-started-multidb/tutorial_model/``::
+
+  Model: multi shard logger@00-00-00:The colocated db tensor is: [1 2 3 4]
+  Model: single shard logger@00-00-00:The clustered db tensor is: [5 6 7 8]
+
+This output showcases that we have established a connection with multiple Orchestrators.
+
+Next, let's take the tensors retrieved from the standard deployment databases and
```

Comment:
remove the "let's" to keep it consistent with the voice the other instructions are written in?